x-pack/plugins/remote_clusters/__jest__/client_integration/list/remote_clusters_list.test.js

```diff
@@ -442,6 +451,11 @@ describe('<RemoteClusterList />', () => {
         actions.clickRemoteClusterAt(1); // the remoteCluster2 has been configured by node
         expect(exists('remoteClusterConfiguredByNodeWarning')).toBe(true);
       });
+
+      test('Should display authentication type', () => {
+        actions.clickRemoteClusterAt(2);
+        expect(exists('authenticationTypeLabel')).toBe(false);
```

Comment:
nit: Shouldn't this check if the label **does** exist?
```suggestion
        expect(exists('authenticationTypeLabel')).toBe(true);
```